### DISCUSSION TITLE
covert kodak PNG images into JP2 with sampling factor

### DISCUSSION THREAD

**Discussion by khawar-islam** [ImageMagick/ImageMagick#2727] [Help]
I am trying to convert png images into JP2 with sampling factor. I am imagemagick tool for conversion, as they mentioned in docs -sampling-factor is used for sampling but it did not work. 

http://ftp.icm.edu.pl/pub/unix/graphics/ImageMagick/www/jp2.html

convert wizard.png -colorspace YUV -sampling-factor 2,2 wizard.jp2

My command, it did not work
convert test/images/kodim$i.png -quality $(($j*5)) -colorspace YUV -sampling-factor 4,2 test/JP2/kodim$i/`printf "%02d" $j`.jp2

**Comment by urban-warrior** [comment]
We need more information to help with this problem.  How did the command fail?  What is the exception message you received?
